lib/src/breadcrumb/breadcrumb.component.html

```diff
@@ -0,0 +1,14 @@
+<div class="custom-breadcrumb" [ngClass]="className" [attr.id]="id">
+    <nav aria-label="breadcrumb">
+        <ol class="breadcrumb">
+            <ng-container *ngFor="let breadcrumb of list; index as i">
+                <li
+                    class="breadcrumb-item"
+                    [ngClass]="{ active: i === list.length - 1 }"
+                    (click)="handleClick(i, $event)"
+                    [innerHTML]="breadcrumb | safeHtml"
```

Comment:
just wondering why the need for the innerHTML using safeHtml here? In case someone wants to pass custom html instead of just label?

Reply:
Yah the user can pass either .